enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/controller/api/IdentityVerificationController.java

```diff
@@ -367,7 +380,8 @@ public ObjectResponse<DocumentVerificationSdkInitResponse> initVerificationSdk(
         final OwnerId ownerId = PowerAuthUtil.getOwnerId(apiAuthentication);
         final String processId = request.getRequestObject().getProcessId();
 
-        onboardingService.verifyProcessId(ownerId, processId);
+        // Onboarding process is not locked
```

Comment:
What about to transfer these comments to DEBUG or TRACE logger?

Reply:
I will move them.